web-console/3-project-details/deployment-details.md

```diff
@@ -0,0 +1,42 @@
+# Deployments: Details Page
+
+- Selecting a deployment from the Deployments landing page or from the Overview will bring the user to the Deployment Details page.
+- The details page for Deployments will include four tabs: History, Configuration, Environment, Events.
+
+## History Tab
+No change at this time.
+
+## Configuration Tab
+No change at this time.
+
+## Environment Tab
+ ![template](img/v2-EnvVar-05.png)
+- This tab will now consist of two parts, (1) the "Variables" section that already existed and (2) the "Environment From" section that is new to the page.
+- These grey (#757575) subheadings will be used to separate the two sections.
+- Once any changes have been made in either area, the Save button should become enabled as a primary action button (in blue).
+
+#### Variables
+- This top section lets the user create new environment variables or create variables by picking one specific key from a secret or config map.
+- Order is important and users can sort variables within this section.
+- The two action links should be called "Add Value" and "Add Value from a Resource."
+
```

Comment:
I'm still not sure about the terminology here. You're really adding a variable in this section. "Value" is part of what you're adding. And "resource" is vague to me. It's really just two types of resources that you can add values from. 

What's there for those actions today is long, but it is also more accurate and specific. What do you think, @spadgett ?

Reply:
I'm OK with 

Add Variable | Add Variable From Config Map or Secret

I think that's what you're suggesting @cshinn / @ncameronbritt ?